src/capsule/styles.css

```
@@ -188,6 +190,15 @@ button {
   color: var(--capsule-error);
 }
 
+.terminal-dictionary {
+  color: var(--capsule-primary);
+}
+
+.terminal-dictionary svg {
+  color: var(--capsule-primary) !important;
+  fill: currentColor !important;
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)